graph_altimeter/scan/__init__.py

```diff
@@ -155,3 +161,26 @@ def remove_scan_files(temp_dir, scan_id):
         shutil.rmtree(scan_path)
     except FileNotFoundError:
         logger.debug("temp directory %s not found", scan_path)
+
+
+class InvalidAWSAccount(Exception):
+    """Returned when normalized an AWS account identifier if the format is not
+    recognized."""
+
+    def __init__(self, AWS_account):
```

Comment:
s/AWS_account/aws_account/ ?

I feel mixing cases in parameters is not very pythonish.

Reply:
agree it isn't.